.changeset/many-sloths-shop.md

```diff
@@ -0,0 +1,15 @@
+---
+"graphql-language-service-server": patch
+"graphql-language-service": patch
+---
```

Comment:
I'm unclear if I need to list vscode-graphql or anything else to get the vscode extension updated?